fastn-builtins/src/lib.rs

```diff
@@ -1036,6 +1036,52 @@ pub fn default_bag() -> indexmap::IndexMap<String, fastn_resolved::Definition> {
                 external_implementation: true
             })
         ),
+        (
+            "ftd#app-path-ex".to_string(),
```

Comment:
I was thinking you make the `app` a required field and only expose one function.

Reply:
I've removed `app-path-ex` form builtins. `app_path_ex` in ftd.js file will be internally called. The `app` param is still optional. If it's not passed then we take the current app from the context (i.e, if the file is present in lets-auth package then app = "lets-auth"). Do you want me to make this explicit? I kinda like this implicitness, especially when you consider that, as an app author, your code will be filled with such function calls where you're constructing urls for your app but you still have to pass your app name every time.

Reply:
No, the behaviour you describe is perfect, I like optional app as well, and the general pattern of optional function values (and optional component arguments etc) for fastn ecosystem.